.github/workflows/release.yml

```diff
@@ -216,6 +155,49 @@ jobs:
           if [ "${{ inputs.dry_run }}" = "true" ]; then  
             box_summary="[Dry run] "
           fi
+            echo "$box_summary :ballot_box: ${{ inputs.release_version }} was uploaded to https://pentaho.app.box.com/folder/$BOX_FOLDER_ID" >> $GITHUB_STEP_SUMMARY
+
+
+
+      - name: Promote artifacts
+        if: ${{ inputs.promote_artifacts == true }}
+        run: |
+          rt_repo='pntpub-maven-release'
+
+          if [ "${{ env.REPOSITORY_IS_PRIVATE }}" = "true" ]; then
+            rt_repo='pntprv-maven-release'
+          fi
+
+          promotion_summary=""
+          if [ "${{ inputs.dry_run }}" = "true" ]; then
+            promotion_summary="[Dry run] "
+          fi
+          jf rt build-promote "${{ env.BUILD_NAME }}" "${{ env.BUILD_NUMBER }}" ${rt_repo} --dry-run=${{ inputs.dry_run }}
           
-          # BOX_FOLDER_ID is an env var that is set inside the promote-release-to-box.py script file
-          echo "$box_summary :ballot_box: ${{ inputs.release_version }} was uploaded to https://pentaho.app.box.com/folder/$BOX_FOLDER_ID" >> $GITHUB_STEP_SUMMARY
+          echo "$promotion_summary :frog: Version ${{ inputs.release_version }} was promoted to ${rt_repo}" >> $GITHUB_STEP_SUMMARY 
+        
+      - name: Create Release Bundle
+        if: ${{ inputs.create_release_bundle == true }}
+        shell: bash
+        run: |
+          python3 -m venv env
+          source env/bin/activate
+          pip3 install -q pyyaml dohq-artifactory requests
+          echo ${{ env.BUILD_NAME }} ${{ env.BUILD_NUMBER }} ${{ env.BUILD_VERSION }}  
+          python3 actions-common/.github/scripts/python/release-bundle.py \
+          --build_name ${{ env.BUILD_NAME }} \
+          --build_number ${{ env.BUILD_NUMBER }} \
+          --build_version  ${{ env.BUILD_VERSION }} \
+          --rt_auth_username ${{ secrets.PENTAHO_CICD_ONE_USER }} \
+          --rt_auth_password ${{ secrets.PENTAHO_CICD_ONE_KEY }} \
+          --manifest_file_path ${{ env.MANIFEST_PATH}} \
+          --rt_base_url ${{ env.ARTIFACTORY_BASE_URL }} \
+          --jf_cli_rt_name ${{ inputs.jf_cli_rt_name }} \
+          --dry_run ${{ inputs.dry_run }} \
+          --logging_level ${{inputs.logging_level }} \
+          --release_bundle_name ${{ env.BUILD_NAME }} \
+          --release_bundle_version ${{ env.BUILD_NUMBER }} \
+          --signing_key_name "frog" \
+          --arti_host ${{ env.ARTIFACTORY_HOST }} \
+          --release_method "artifacts"
+           
```

Comment:
Still seeing the some white space in the last line (posting image)
![image](https://github.com/pentaho/actions-common/assets/13484152/83c938be-321e-43c2-ad58-47a1e9878d6a)